### ISSUE TITLE
-webkit-radial-gradient isn't added

### ISSUE THREAD

**Issue by danfisher85** [postcss/autoprefixer#854]
Looks like -webkit-radial-gradient is not added after last release.

## Input
`background-image: radial-gradient(circle, at top left, rgba(255,255,255,.8), transparent);`

## Output
`background-image: radial-gradient(circle, at top left, rgba(255,255,255,.8), transparent);`

## Expected
```
background-image: -webkit-radial-gradient(top left, circle, rgba(255, 255, 255, 0.8), transparent);
background-image: radial-gradient(circle, at top left, rgba(255,255,255,.8), transparent);
```

## Browsers List
`> 1%, last 2 versions, Firefox ESR, Opera 12.1`

**Comment by ai** [comment]
https://github.com/postcss/autoprefixer#why-doesnt-autoprefixer-add-prefixes-to-border-radius

According Can I Use we don’t need this prefix for this browsers: http://caniuse.com/#search=gradients

**Comment by frandiox** [comment]
@ai MDN says `-webkit` prefix is necessary on `background-image`: https://developer.mozilla.org/en-US/docs/Web/CSS/radial-gradient#Browser_compatibility
I just tested on Chrome and Safari and looks like both do need `-webkit` prefix :/

**Comment by ai** [comment]
@frandiox please show input CSS, output CSS and expected output.

**Comment by bognarl** [comment]
@frandiox This is still a problem.

My input scss is:
`background-image: radial-gradient(50% 70%, ellipse cover, white, color(grayscale,base) 100%);`

Output css:
`background-image: radial-gradient(50% 70%, ellipse cover, white, #b7b7b7 100%);`

Tried with this too:
```
    background-image: -webkit-radial-gradient(50% 70%, ellipse cover, white, color(grayscale,base) 100%);
    background-image: radial-gradient(50% 70%, ellipse cover, white, color(grayscale,base) 100%);
```
Same output

**Comment by frandiox** [comment]
@bognarl Set `remove` option to `false` so it doesn't remove the prefixes you added manually. In my case, due to my project I've set `browsers` to `['> 1%', 'Firefox ESR', 'Opera 12.1', 'Android >= 4.4', 'iOS >= 8.0', 'Chrome >= 30', 'Safari >= 9',]` and it works well. I think `> 1%` and `iOS >= 8.0` were necessary for `-webkit-radial-gradient`. It is weird that Chrome requires it but canuse doesn't show it.

**Comment by bognarl** [comment]
@frandiox Thanks I've just edited my comment. Reverse order works perfectly no need any other modification in the options.

**Comment by ai** [comment]
@bognarl you use old direction syntax. This is why unprefixed version doesn't work (it uses only new direction syntax).

Latest Autoprefixer shows warning about old direction.

**Comment by frandiox** [comment]
@ai I just discovered why it wasn't working in my case even though it was new direction syntax. The problem was the `cover` keyword, which according to MDN:

>Note: Early implementations of this function included other keywords (`cover` and `contain`) as synonyms of the standard `farthest-corner` and `closest-side`, respectively. Use the standard keywords only, as some implementations have already dropped those older variants.

Removing it or using another valid keyword makes it work in Chrome without `-webkit` prefix. I don't see any notification from autoprefixer in this specific situation, not sure if you want to add one. Thanks!

**Comment by ai** [comment]
@frandiox hm, maybe I will add warning for this keyword too.

**Comment by ai** [comment]
@frandiox I added extra warning for your case eb11f02

**Comment by frandiox** [comment]
@ai Awesome, thanks :)

**Comment by ai** [comment]
Warning was released in 7.2
